mesa/visualization/solara_viz.py

```diff
@@ -411,6 +411,19 @@ def _check_model_params(init_func, model_params):
         ValueError: If a parameter is not valid for the model's initialization function
     """
     model_parameters = inspect.signature(init_func).parameters
+
+    has_var_positional = any(
```

Comment:
@nissu99 If we have to use keyword arguments for solara so it updates when the parameter changes, shouldn't this throw an error is someone uses a positional argument?

Reply:
i think right now it takes postional arguments too, we might need to make the changes in the model itself to strictly handle keyword only args, please tell if i am wrong, thank you .